src/rosdiscover/observer/ros1.py

```diff
@@ -50,7 +47,7 @@ def _summarise(self, contexts: Collection[NodeContext]) -> SystemSummary:
         node_to_summary = {s.fullname: s for s in summaries}
         return SystemSummary(node_to_summary)
 
-    def _transform_state_to_nodeinfo(self, ros: ROS1) -> Collection[NodeInfo]:
+    def _transform_state_to_nodeinfo(self, info: SystemState) -> Collection[NodeInfo]:
```

Comment:
Should `info` be `state` to avoid confusion with `nodeinfo`?

Reply:
Done.